angular-demo/tsconfig.base.json

```
@@ -11,7 +11,7 @@
     "experimentalDecorators": true,
     "moduleResolution": "node",
     "importHelpers": true,
-    "target": "es2015",
+    "target": "ES2022",
     "module": "es2020",
```

Shouldn't it be changed?

```suggestion
    "module": "ES2022",
```